cms/static/js/views/pages/container.js

```diff
@@ -176,6 +176,10 @@ function($, _, Backbone, gettext, BasePage,
                         break;
                     case 'scrollToXBlock':
                         document.getElementById(data.payload.locator)?.scrollIntoView({behavior: "smooth"});
+                        // This piece of code helps to avoid clipping the IFrame when scrollIntoView is triggered.
+                        setTimeout(() => {
+                          window.scrollBy(0, -80); // scroll up to avoid clipping
+                        }, 400); // wait for scroll to complete
```

Comment:
Nit: Can we simplify the comments reduce the size of this diff? Seems too verbose.